Sources/StreamChatUI/Common Views/CurrentChatUserAvatarView_Tests.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class CurrentChatUserAvatarView_Tests: XCTestCase {
+    var mockedCurrentUserController: CurrentChatUserController_Mock<NoExtraData>! = .mock()
+
+    func test_customApperanceHook_currentChatUserAvatarView_withImage_looksAsExpected() {
+        class TestAvatarView: CurrentChatUserAvatarView {}
+
+        TestAvatarView.defaultAppearance {
+            $0.avatarView.imageView.isOpaque = true
+            $0.backgroundColor = .brown
+            $0.avatarView.imageView.backgroundColor = .cyan
+        }
+
+        mockedCurrentUserController.currentUser_mock = .init(
+            id: "Yoda",
+            imageURL: TestImages.yoda.url
+        )
```

Comment:
this is kind of code that should be in `setUp() { }`

Reply:
Yep, thanks done :)